app/main/templates/base.html

```diff
@@ -8,8 +8,13 @@
         <title>{% block title %}hazen-web-app{% endblock %}</title>
 
         {% block styles %}
-            <!-- Bootstrap CSS -->
-            {{ bootstrap.load_css() }}
+            <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
+                  rel="stylesheet"
+                  integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN"
+                  crossorigin="anonymous">
+            <link href="https://fonts.googleapis.com/css?family=Inter&display=swap" rel="stylesheet" />
+            <link href="https://fonts.googleapis.com/css?family=Monda&display=swap" rel="stylesheet" />
+            <link href="/static/css/hazen.css" rel="stylesheet" type="text/css" />
```

Comment:
please add comments for what these links are for?

Reply:
Done